param/reactive.py

```diff
@@ -235,6 +287,27 @@ def pipe(self, func, /, *args, **kwargs):
         """
         return self._as_rx()._apply_operator(func, *args, **kwargs)
 
+    def resolve(self, nested=True, recursive=False):
```

Comment:
```suggestion
    def resolve(self, *, nested=True, recursive=False):
```

So that we don't get a function call like this resolve(True, True)